pkg/internal/codegen/parse/util.go

```diff
@@ -233,8 +233,8 @@ func hasSingular(t *types.Type) bool {
 
 // hasRequired returns true if t is annotated with
 // +required
-func hasRequired(t *types.Type) bool {
-	for _, c := range append(append([]string{}, t.SecondClosestCommentLines...), t.CommentLines...) {
+func hasRequired(t types.Member) bool {
+	for _, c := range t.CommentLines {
```

Comment:
intentional?

Reply:
@sttts yeah, `Member`s of a `Type` don't have `SecondClosestCommentLines`

Reply:
Interesting. But looks like that.